### PR TITLE
Split into the submodules

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] split code
- [x] rewrite help
- [x] rewrite README
- [ ] ~~polish tooling (therapist, CircleCI etc.)~~

### CODE REVIEW

**Comment by leplatrem** [comment]
For all the script/circleCI, I think I'll let you do it @peterbe, since you've gone through this recently... What do you think?

File: aws_lambda.py [hunk 1]
```diff
@@ -1,53 +1,23 @@
...
-@command
 def help(**kwargs):
     """Show this help.
     """
 
     def white_bold(s):
         return f"\033[1m\x1B[37m{s}\033[0;0m"
 
+    entrypoints = [
+        f.rsplit("/")[-1].replace(".py", "") for f in glob.glob("./commands/[a-z]*.py")
```
Comment (peterbe) [inline aws_lambda.py]: Why the `[a-z]`? What if I want to call it "Stuff.py" or "check-redash.py" or "check_temp.py"?

Comment (peterbe) [inline aws_lambda.py]:
```suggestion
        os.path.splitext(os.path.basename(f))[0] for f in glob.glob("./commands/[a-z]*.py")
```

Comment (leplatrem) [reply aws_lambda.py]: just to exclude `__init__.py`. And also, you won't want `Stuff.py`. 

`check_temp.py` is fine

File: aws_lambda.py [hunk 2]
```diff
@@ -1,53 +1,23 @@
...
-@command
 def help(**kwargs):
     """Show this help.
     """
 
     def white_bold(s):
         return f"\033[1m\x1B[37m{s}\033[0;0m"
 
+    entrypoints = [
+        os.path.splitext(os.path.basename(f))[0] for f in glob.glob("./commands/[a-z]*.py")
```
Comment (peterbe) [inline aws_lambda.py]:
```suggestion
        os.path.splitext(os.path.basename(f))[0]
        for f in glob.glob("./commands/[a-z]*.py")
```

for `black` to be happy.

File: aws_lambda.py (hunk 1 above)
Comment (peterbe) [reply aws_lambda.py]: Oh. I get it. Weird. POSIX vs. RegExp I guess.

**Comment by peterbe** [comment]
It's a bit off-topic but this is a bit "ugly":

```
▶ SERVER=https://firefox.settings.services.mozilla.com/v1/  python aws_lambda.py --help
Unknown function '--help'
Traceback (most recent call last):
  File "aws_lambda.py", line 51, in <module>
    mod = importlib.import_module(f"commands.{entrypoint}")
  File "/Users/peterbe/virtualenvs/remote-settings-lambdas/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'commands.--help'
```

**Comment by peterbe** [comment]
Also a bit off-topic but mine's failing because, I suspect, my `kinto_http` isn't up-to-date:

```
▶ SERVER=https://firefox.settings.services.mozilla.com/v1/  python aws_lambda.py help
Traceback (most recent call last):
  File "aws_lambda.py", line 47, in <module>
    help()
  File "aws_lambda.py", line 21, in help
    for entrypoint in entrypoints
  File "aws_lambda.py", line 21, in <listcomp>
    for entrypoint in entrypoints
  File "/Users/peterbe/virtualenvs/remote-settings-lambdas/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/peterbe/dev/MOZILLA/REMOTE_SETTINGS/remote-settings-lambdas/commands/backport_records.py", line 3, in <module>
    from kinto_http import Client, BearerTokenAuth
ImportError: cannot import name 'BearerTokenAuth' from 'kinto_http' (/Users/peterbe/virtualenvs/remote-settings-lambdas/lib/python3.7/site-packages/kinto_http/__init__.py)
(remote-settings-lambdas)
MOZILLA/REMOTE_SETTINGS/remote-settings-lambdas  (81d3e70...) ✔
▶ python
Python 3.7.2 (default, Jan 31 2019, 16:22:44)
[Clang 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import kinto_http
>>> from kinto_http import Client, BearerTokenAuth
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'BearerTokenAuth' from 'kinto_http' (/Users/peterbe/virtualenvs/remote-settings-lambdas/lib/python3.7/site-packages/kinto_http/__init__.py)
```

**Comment by peterbe** [comment]
Hmmm... See:

```
MOZILLA/REMOTE_SETTINGS/remote-settings-lambdas  (81d3e70...) ✔
▶ rg kinto-http
requirements.pip
6:kinto-http>=10.2.0

requirements.txt
84:kinto-http==10.2.0 \
(remote-settings-lambdas)
MOZILLA/REMOTE_SETTINGS/remote-settings-lambdas  (81d3e70...) ✔
▶ pip list | rg kinto-http
kinto-http      10.2.0
```

The `requirements.pip` suggests `kinto-http>=10.2.0` so you should get https://pypi.org/project/kinto-http/10.3.0/ right? 
Is the hardcoded `kinto-http==10.2.0` inside `requirements.txt` holding it back? Sure seems so.

**Comment by peterbe** [comment]
I think the above-mentioned points are unrelated. It works:

```
▶ SERVER=https://firefox.settings.services.mozilla.com/v1/  python aws_lambda.py help

Remote Settings lambdas.

Available commands:

 - blockpages_generator: Generate the blocklist HTML pages and upload them to S3.

 - validate_changes_collection: Validate the changes monitor endpoint entries.

 - backport_records: Backport records creations, updates and deletions from one collection to another.

 - consistency_checks: Check the collections content and status consistency.

 - validate_signature: Validate the signature of each collection.

 - refresh_signature: Refresh the signatures of each collection.
```

The only "bad" thing is the cryptic sentence:
```
Validate the changes monitor endpoint entries
```

**Pull request merged**